src/checkers/inference/solver/util/Constants.java

```diff
@@ -0,0 +1,26 @@
+package checkers.inference.solver.util;
+
+/**
+ * String constants.
+ * 
+ * @author jianchu
+ *
+ */
+public class Constants {
```

Comment:
Which kind of organization should we use? Should we separate those categories into different class, or we put those enums in a single class to manage? Which one is better?